server/controllers/aidControllers.js

```diff
@@ -0,0 +1,127 @@
+import { INTERNAL_SERVER_ERROR,CREATED,OK,NOT_FOUND,BAD_REQUEST } from 'http-status-codes';
+const db = require("./promise").AidDb;
+import validateAidQueryText from '../validation/aid'
+
+
+const Aids = {
+    async create(req, res) {
+        const { errors, isValid } = validateAidQueryText(req.body);
+
+		// Check Validation
+		if (!isValid) {
+			return res.status(BAD_REQUEST).send({
+                status : "fail",
+                data : { "title" : errors }
+            });
+		}
+        const queryText = req.body;
+        try {
+          const aid = await db.create(queryText);
+          return res.status(CREATED).send({
+            status: 'success',
+            data: { aid, message: 'Aid successfully created' }
+          });
+        } catch (error) {
+          return res.status(INTERNAL_SERVER_ERROR).send({
+            status : 'error',
+            message : error
+          });
+        }
+    },
+
+	async getAll(req, res) {
+        try {
+            const aids = await db.find();
+            return res.status(OK).send({
+                status: 'success',
+                data: {"Aids": aids}
```

Comment:
In javascript first letters of keys in objects are lower case => `data: { aids }`